includes/im_user_userheader_block.inc

```diff
@@ -48,18 +46,18 @@ function _im_user_userheader_bar() {
   $timestamp =strtotime($timestamp);
   $datedisplay = format_date($timestamp,'custom','l j F Y');
   $timedisplay = array('class'=>'im_user_portal_time');
-  $item_display[0] = $datedisplay;
-  $item_display[] = $timedisplay;
+  $item_display[0] = '';
+  //$item_display[] = $timedisplay;
   $count = 0;
-if(!empty ($user_fields->field_user_stores['und'][0]['value'])) {
-    $prefered_store = $user_fields->field_user_stores['und'][0]['value'];
+  if (!empty ($user_fields->field_user_stores['LANGUAGE_NONE'][0]['value'])) {

```

Comment:
I would remove the quotes in LANGUAGE_NONE key mentioned and re check the same and push the code. I agree quality is the preference than speed.